README.md

```diff
@@ -28,7 +28,9 @@ The factory accepts the following options:
 - `factory`: The factory used to create Dataset instances. Default: `require('rdf-ext')`
 - `formats`: An object with `parsers` and `serializers`, each given as `@rdfjs/sink-map`. Default: `require('@rdfjs/formats-common')`
 - `defaultMediaType`: If an unknown Content-Type is given, this media type will be used. Default: `undefined`
-- `baseIRI`: If `true`, will call [absolute-url](https://npm.im/absolute-url) to get the requested IRI and pass as base IRI to the parser. It can also be a function `async (req) => string` which can be used to compute the base IRI for the parser.
+- `baseIriFromRequest`: If `true`, will call [absolute-url](https://npm.im/absolute-url) to get the requested IRI and pass as base IRI to the parser.
+  It can also be a function `async (req) => string` which can be used to compute the base IRI for the parser.
+- `toTriple`: If `true`, the RDF/JS Quads sent using `res.dataset()` or `res.quadStream()` are converted to triples. Default `undefined`.
```

Comment:
Oh, it's a setting of the handler. I thought it would be an explicit method or parameter on `res.dataset()`/`res.quadStream()`

Are you sure that the default should not be the opposite? I expect triples (default graph) to be the more natural. You said it yourself, a resource representation is essentially a graph itself.

```suggestion
- `respondQuads`: If `true`, the RDF/JS Quads sent using `res.dataset()` or `res.quadStream()` are not converted to triples (default graph). Default `false`.
```

Reply:
I think there would be not to much benefit adding it as an option to the `res.` methods. The difference between `res.dataset(unsetGraph(dataset))` and `res.dataset(dataset, { toTriple })` would not be to big and redundant for every request handler.

Reply:
No no, I agree. I just mentioned that I misunderstood earlier.

Reply:
I prefer having a default logic that is as simple as possible and enables additional data processing by explicitly setting options. Anyway, I hope people will use higher-level libs like hydra-box or maybe some clownface middleware that takes care of these things.

Reply:
> Anyway, I hope people will use higher-level libs like hydra-box or maybe some clownface middleware that takes care of these things.

Ah, fair enough.

So would you agree to have `hydra-box` flatten the dataset to default graph by default?